charts/kserve-resources/templates/clusterstoragecontainer.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: "serving.kserve.io/v1alpha1"
+kind: ClusterStorageContainer
+metadata:
+  name: default
+spec:
+  storageContainer:
+    name: storage-initializer
+    image: kserve/storage-initializer:latest
+    resources:
+      requests:
+        memory: 100Mi
+        cpu: 100m
+      limits:
+        memory: 1Gi
+        cpu: "1"
+  supportedPrefixes:
+    - 'gs://'
+    - 's3://'
+    - 'hdfs://'
+    - 'webhdfs://'
+    - 'pvc://'
+    - 'file://'
```

Comment:
this is not supported, only for local testing not for cloud environment

Reply:
Do you mean "file://" is not supported?